pod-operation/src/components/gyro.rs

```diff
@@ -1,3 +1,4 @@
+pub const PI: f32 = 3.14159265358979323846264338327950288_f32; // 3.1415926535897931f64
```

Comment:
Chore: we can use constants from [`std::f32::consts`](https://doc.rust-lang.org/std/f32/consts/index.html) rather than defining our own.